src/Main.java

```diff
@@ -1,7 +1,92 @@
 public class Main {
     public static void main(String[] args) {
-        var box = 30;
-        System.out.println(box);
+int bigStock = 2000000;
+System.out.println("Большой склад вмещает " + bigStock + " кг ");
+byte box = 127;
+System.out.println("Коробочка вмещает " + box + " кг ");
+short chest = 32767;
+System.out.println("Сундук вмещает " + chest + " кг ");
+long bigHangar = 11986536269565L;
+System.out.println("Большой ангар вмещает " + bigHangar + " кг ");
+float pieceOfCake = 2.15f;
+System.out.println("Кусочек пирога весит " + pieceOfCake + " кг ");
+double exactValue = 3.7896;
+System.out.println("Точный вес продукта составляет " + exactValue + " кг ");
+float secondPieceOfCake = 27.12f;
+System.out.println("Второй КУСОЧЕК пирога весит " + secondPieceOfCake + " кг ");
+long residents = 987678965549L;
+System.out.println("Всего жителей в городе " + residents);
+float a = 2.786f;
+System.out.println(a);
+short b = 569;
+System.out.println(b);
+short c = -159;
+System.out.println(c);
+short d = 27897;
+System.out.println(d);
+byte e = 67;
+System.out.println(e);
+short totalPepper = 480;
+byte lyudaStudents = 23;
+byte annaStudents = 27;
+byte katyaStudents = 30;
+byte totalStudents = (byte) (lyudaStudents + annaStudents + katyaStudents);
+System.out.println("На каждого ученика рассчитано " + totalPepper / totalStudents + " листов бумаги");
+byte machineWork = 8;
```

Comment:
```suggestion
byte machineWork = 16 / 2;
```
ничего не считаем в уме, всё считает программа пусть

Reply:
и лучше назвать более понятно, например, performancePerMinute